configs/tasks/rearrange/rearrange_easy_multi_agent.yaml

```diff
@@ -0,0 +1,172 @@
+    # from `UsesRobotInterface` then it is automatically duplicated.
+    # Hetrogeneous observation spaces are also supported by not deriving from
+    # `UsesRobotInterface`.
+    SENSORS:
+    - RELATIVE_RESTING_POS_SENSOR
+    - TARGET_START_SENSOR
+    - GOAL_SENSOR
+    - JOINT_SENSOR
+    - IS_HOLDING_SENSOR
+    - END_EFFECTOR_SENSOR
+    - TARGET_START_GPS_COMPASS_SENSOR
+    - TARGET_GOAL_GPS_COMPASS_SENSOR
+    FORCE_TERMINATE:
+        TYPE: ForceTerminate
+        MAX_ACCUM_FORCE: 100_000.0
+        MAX_INSTANT_FORCE: 10_000.0
+    COMPOSITE_SUCCESS:
+        MUST_CALL_STOP: False
+    # Unlike sensors, measurements are not automatically duplicated per agent.
+    MEASUREMENTS:
+    - OBJECT_TO_GOAL_DISTANCE
+    - ROBOT_FORCE
+    - FORCE_TERMINATE
+    - ROBOT_COLLS
+    - COMPOSITE_SUCCESS
+    - NUM_STEPS
+    ACTIONS:
+      # Actions are defined per agent. Unlike the sensors, actions are not
+      # automatically duplicated per agent.
+      AGENT_0_ARM_ACTION:
+          AGENT: 0
+          TYPE: ArmAction
+          ARM_CONTROLLER: ArmRelPosAction
+          GRIP_CONTROLLER: MagicGraspAction
+          ARM_JOINT_DIMENSIONALITY: 7
+          GRASP_THRESH_DIST: 0.15
+          DISABLE_GRIP: false
+          DELTA_POS_LIMIT: 0.0125
+          EE_CTRL_LIM: 0.015
+      AGENT_0_BASE_VELOCITY:
+          AGENT: 0
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_1_ARM_ACTION:
+          AGENT: 1
+          TYPE: ArmAction
+          ARM_CONTROLLER: ArmRelPosAction
+          GRIP_CONTROLLER: MagicGraspAction
+          ARM_JOINT_DIMENSIONALITY: 7
+          GRASP_THRESH_DIST: 0.15
+          DISABLE_GRIP: false
+          DELTA_POS_LIMIT: 0.0125
+          EE_CTRL_LIM: 0.015
+      AGENT_1_BASE_VELOCITY:
+          AGENT: 1
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_0_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+      AGENT_1_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+    POSSIBLE_ACTIONS:
+      - AGENT_0_ARM_ACTION
+      - AGENT_0_BASE_VELOCITY
+      - AGENT_0_REARRANGE_STOP
+      - AGENT_1_ARM_ACTION
+      - AGENT_1_BASE_VELOCITY
+      - AGENT_1_REARRANGE_STOP
+SIMULATOR:
+    ADDITIONAL_OBJECT_PATHS:
+        - "data/objects/ycb/configs/"
+    ACTION_SPACE_CONFIG: v0
+    GRASP_IMPULSE: 10000.0
+    HOLD_THRESH: 0.2
+    AC_FREQ_RATIO: 4
+    DEBUG_RENDER: false
+    DEBUG_RENDER_GOAL: true
+    AGENTS:
+    - AGENT_0
+    - AGENT_1
+    AGENT_0:
+        HEIGHT: 1.5
+        RADIUS: 0.3
+        IS_SET_START_STATE: false
+        SENSORS:
+        - HEAD_DEPTH_SENSOR
```

Comment:
Do we need to add comment here as well that sensors for AGENT_1 will be just copied?